src/main/java/mekanism/common/item/ItemModule.java

```diff
@@ -46,9 +49,17 @@ public Rarity getRarity(@Nonnull ItemStack stack) {
     @Override
     public void appendHoverText(@Nonnull ItemStack stack, Level world, @Nonnull List<Component> tooltip, @Nonnull TooltipFlag flag) {
         if (MekKeyHandler.isKeyPressed(MekanismKeyHandler.detailsKey)) {
+            tooltip.add(new TextComponent("Supported by:"));
             for (Item item : MekanismAPI.getModuleHelper().getSupported(getModuleData())) {
                 tooltip.add(item.getName(new ItemStack(item)));
             }
+            Set<ModuleData<?>> conflicting = MekanismAPI.getModuleHelper().getConflicting(getModuleData());
```

Comment:
Might as well put `MekanismAPI.getModuleHelper()` in a local variable given there are now multiple calls to it in this method.